newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -116,6 +116,13 @@ export default class SearchPanel extends PureComponent<Props, State> {
               }
               hintText={t`Text to search in parameters`}
               onChange={(e, searchText) => this.setState({ searchText })}
+              onKeyPress={event => {
+                if (event.charCode === 13) {
```

Comment:
Use event.code to check for "Enter", and you can then remove the comment that is not useful anymore :)